### PR TITLE
Update to Django 4.1

### PR DESCRIPTION
**Pull request by jacklinke**
- Update tox, setup, & readme to show django 4.1 support
- Update package version, per release guidelines
- Include changes in updated changelog

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -22,4 +22,5 @@ deps =
     django31: Django>=3.1,<3.2
     django32: Django>=3.2,<3.3
     django40: Django>=4.0,<4.1
+    django40: Django>=4.1,<4.2
```
Comment (clokep) [inline tox.ini:25]: Looks like a copy & paste error here:

```suggestion
    django41: Django>=4.1,<4.2
```

Comment (jacklinke) [reply tox.ini]: Oops, good catch

File: CHANGELOG.rst [hunk 2]
```diff
@@ -3,8 +3,8 @@
 Changelog
 #########
 
-next
-====
+0.9.2 (October 13, 2022)
```
Comment (clokep) [inline CHANGELOG.rst:6]: I'll do a release today.

```suggestion
0.9.2 (October 14, 2022)
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -15,6 +15,7 @@ Maintenance
   and `flake8 <https://flake8.pycqa.org>`_, and `pyupgrade <https://github.com/asottile/pyupgrade>`_.
   (`#38 <https://github.com/clokep/django-render-block/pull/38>`_,
   `#39 <https://github.com/clokep/django-render-block/pull/39>`_)
+* Update to include and run tests for Django 4.1
```
Comment (clokep) [inline CHANGELOG.rst:18]:
```suggestion
* Update to include and run tests for Django 4.1. Contributed by
  `Jack Linke <https://github.com/jacklinke>`_.
    (`#41 <https://github.com/clokep/django-render-block/pull/41>`_)
```

**Review by clokep — APPROVED** [review]
Thank you!

**Comment by clokep** [comment]
Sorry for the delay, I've put up a release with this!

**Pull request merged**